packages/embed/vite.config.js

```diff
@@ -56,7 +53,12 @@ export default defineConfig(({ command, mode }) => {
           }
         }
       },
-      react(),
+      react({
```

Comment:
took me a while to figure out we needed this to support custom css prop in harmony. the docs spell it out nicely https://github.com/AudiusProject/audius-protocol/tree/main/packages/harmony#setup